### PR TITLE
enable ansible repo on el8

### PR DESCRIPTION
**Pull request by jlsherrill**
Cherry-pick into:

* [ ] Foreman 2.4
* [ ] Foreman 2.3 (Satellite 6.9)
* [ ] Foreman 2.1 (Satellite 6.8)

### CODE REVIEW

**Comment by jlsherrill** [comment]
Context here:  https://github.com/theforeman/pulpcore-packaging/issues/138

**Comment by melcorr** [comment]
@jlsherrill is this ready to go? 
Does it need cherry picking for the release?

**Comment by jlsherrill** [comment]
Should be ready to go, only needed for nightly right now (will be needed for katello4.1/foreman 2.5)

**Review by melcorr — APPROVED** [review]
LGTM thank you!

File: guides/common/modules/proc_configuring-repositories-el.adoc [hunk 1]
```diff
@@ -23,6 +23,16 @@ ifdef::katello[]
 ----
 # {package-manager} localinstall https://yum.theforeman.org/katello/{KatelloVersion}/katello/el{distribution-major-version}/x86_64/katello-repos-latest.rpm
 ----
+
+ifeval::["{distribution-major-version}" == "8"]
++
+. Install the `centos-release-ansible-29` package to enable ansible collection support
```
Comment (evgeni) [inline guides/common/modules/proc_configuring-repositories-el.adoc:29]: "enable ansible collection support" sounds like this repo contains it, which is not correct.
```suggestion
. Install the `centos-release-ansible-29` package to enable repositories for dependencies of the Ansible collection support
```

File: guides/common/modules/proc_configuring-repositories-el.adoc [hunk 2]
```diff
@@ -23,6 +23,16 @@ ifdef::katello[]
 ----
 # {package-manager} localinstall https://yum.theforeman.org/katello/{KatelloVersion}/katello/el{distribution-major-version}/x86_64/katello-repos-latest.rpm
 ----
+
+ifeval::["{distribution-major-version}" == "8"]
```
Comment (evgeni) [inline guides/common/modules/proc_configuring-repositories-el.adoc:27]: this should only happen on CentOS, not RHEL.

File: guides/common/modules/proc_configuring-repositories-el.adoc (hunk 1 above)
Comment (melcorr) [reply guides/common/modules/proc_configuring-repositories-el.adoc:29]: I was too quick. I will fix this now sorry @evgeni

File: guides/common/modules/proc_configuring-repositories-el.adoc (hunk 2 above)
Comment (melcorr) [reply guides/common/modules/proc_configuring-repositories-el.adoc:27]: On testing this only renders for Centos 8

File: guides/common/modules/proc_configuring-repositories-el.adoc (hunk 1 above)
Comment (evgeni) [reply guides/common/modules/proc_configuring-repositories-el.adoc:29]: @jlsherrill is ansible enabled by default on 4.1, or optional? If enabled by default, I'd not explain *why* it's needed at all. We don't for the other repos.

**Pull request merged**